<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeSelector</class>
 <widget class="QDialog" name="NodeSelector">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>489</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select a MEGA folder</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">/********** LEFT MENU **********/
/*ColorTokenIcon;Custom;fCloudDrive;normal;off;colorToken.icon-secondary*/
/*ColorTokenIcon;Custom;fCloudDrive;normal;on;colorToken.icon-primary*/
/*ColorTokenIcon;Custom;fIncomingShares;normal;off;colorToken.icon-secondary*/
/*ColorTokenIcon;Custom;fIncomingShares;normal;on;colorToken.icon-primary*/
/*ColorTokenIcon;Custom;fBackups;normal;off;colorToken.icon-secondary*/
/*ColorTokenIcon;Custom;fBackups;normal;on;colorToken.icon-primary*/
/*ColorTokenIcon;Custom;fRubbish;normal;off;colorToken.icon-secondary*/
/*ColorTokenIcon;Custom;fRubbish;normal;on;colorToken.icon-primary*/
/********** LEFT MENU **********/ 

#wStorage 
{
border:none;
}

#wStorage QFrame
{
    height: 32px;
    padding-right: 6px;
    padding-left: 6px;
}

#wSearchNS  QFrame
{
    padding-right: 6px;
    padding-left: 6px;
}

#cbAlwaysUploadToLocation
{
    spacing: 10px;
}

#wLeftPaneNS
{
    background-color: #F7F7F7; /*colorToken.surface-1*/
}

#wRightPaneHeader, #sCurrentContent
{
    background-color: white;
}

QGroupBox::title
{
    color: #666666;
    padding-left: 10px;
}</string>
  </property>
  <property name="TOKENIZED" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wLeftPaneNS" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="wStorage">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="wSearchLayout_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>24</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="SideBarTab" name="fCloudDrive" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="icon" stdset="0">
            <iconset resource="../../../gui/Resources_common.qrc">
             <normaloff>:/cloud_off</normaloff>
             <normalon>:/cloud_on</normalon>:/cloud_off</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SideBarTab" name="fIncomingShares" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="icon" stdset="0">
            <iconset resource="../../../gui/Resources_common.qrc">
             <normaloff>:/folder-users_off</normaloff>
             <normalon>:/folder-users_on</normalon>:/folder-users_off</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SideBarTab" name="fBackups" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="icon" stdset="0">
            <iconset resource="../../../gui/Resources_common.qrc">
             <normaloff>:/devices_off</normaloff>
             <normalon>:/devices_on</normalon>:/devices_off</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SideBarTab" name="fRubbish" native="true">
           <property name="icon" stdset="0">
            <iconset resource="../../../gui/Resources_common.qrc">
             <normaloff>:/trash_off</normaloff>
             <normalon>:/trash_on</normalon>:/trash_off</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="wSearchNS">
        <property name="title">
         <string>Search</string>
        </property>
        <layout class="QVBoxLayout" name="wSearchLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>24</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item alignment="Qt::AlignBottom">
          <widget class="QFrame" name="fSearchStringNS">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="bSearchNS">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="icon">
               <iconset resource="../../../gui/Resources_common.qrc">
                <normaloff>:/images/node_selector/sidebar/search_default.png</normaloff>:/images/node_selector/sidebar/search_default.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default_icon" stdset="0">
               <string notr="true">qrc:/images/node_selector/sidebar/search_default.png</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tClearSearchResultNS">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../gui/Resources_common.qrc">
                <normaloff>:/images/transfer_manager/sidebar/delete_search_result_default.png</normaloff>:/images/transfer_manager/sidebar/delete_search_result_default.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="default_icon" stdset="0">
               <string notr="true">qrc:/images/transfer_manager/sidebar/delete_search_result_default.png</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SideBarTab</class>
   <extends>QWidget</extends>
   <header>SideBarTab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../gui/Resources_common.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
